### ISSUE TITLE
[BUG]: When many unread messages with the same subject, they are counted as one message

### ISSUE THREAD

**Issue by erropix** [inbasic/ignotifier#227]
When there's many (two or more) unread messages with the same subject, they are counted as one message and I can't delete or archive them.

**Comment by inbasic** [comment]
@trlkly 
> Another problem is that, if someone sends you two emails that are similar, all the similarities will be replaced with "[quoted text removed]."

Is this fixed now? https://github.com/inbasic/ignotifier/issues/311

**Comment by ERROPiX** [comment]
@inbasic the quoted messages was fixed, but not sure about this #227

**Comment by ERROPiX** [comment]
@inbasic, I just want to let you know about some addon that did it better than you, please try it maybe you can get ideas from his code, check the addon here 
https://addons.mozilla.org/en-us/firefox/addon/fastest-notifier-for-gmail/

**Comment by justanotheranonymoususer** [comment]
@ERROPiX seems like it just shows the mobile version of Gmail in a browser control.

**Comment by KissDaniGH** [comment]
there are many of them, but I really don't think that one is better.
https://addons.mozilla.org/en-us/firefox/search/?q=gmail

**Comment by ERROPiX** [comment]
Yes it show the mobile version, but with custom styling because the mobile version do not show good UI in firefox, but what I liked is the ability to do everything gmail do
